announcements/_posts/2020-02-28-w07-fedora-fcaic.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Tim Zabel (@Tjzabel)
+title: "Week #7: Fedora Community Lead"
+layout: post-event
+date-start: "2020-02-28 16:30"
+date-end: "2020-02-28 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey RITlug!
+
+This week is a special meeting. The Fedora Community Action & Impact Coordinator, Marie Nordin, will be coming to give a talk to RITlug this Friday.
```

Comment:
After "Fedora Community Action & Impact Coordinator" put "(FCAIC)" -- it will make the second paragraph flow easier because you aren't second-guessing what that acronym means. 

Other than that, this looks good 👍

Reply:
done.